book/getting_started/system_requirements.md

```diff
@@ -0,0 +1,12 @@
+# System Requirements
+
+## Hardware
```

Comment:
It seems these hardware requirements are tailored towards full nodes, I'd be even clearer about that by stating that in a section header, i.e.:
```markdown
## Hardware
### Archive Node
etc...
```
Want to be super clear about this so that operators understand that these are hardware requirements for a specific type of node and are not turned away

Reply:
I think there's no significant difference in the hardware except for disk space. Do you think we should separate it?

Reply:
Tbd if we add parallel evm in the future id expect memory/cpu to be affected, but for now let's keep as is and focus on iterating